### PR TITLE
Style setup

### PR DESCRIPTION
**Pull request by zrpnr**
Added css and sass loaders to webpack, moved styles out of the index.html.
Did some re-arranging of the template to match the comp I showed @gabesullice today.

Resource links are now in a side menu, everything is fixed height to the viewport and the location is moved into a header.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { render } from 'react-dom';
+import css from './css/main.scss';
```
Comment (gabesullice) [inline src/index.js:3]: Will this PR help load the CodeMirror CSS?

File: webpack.config.js [hunk 2]
```diff
@@ -28,10 +31,37 @@ module.exports = {
...
+              ident: 'postcss',
+              plugins: () => [
+                postcssCustomProperties(),
+                autoprefixer
+              ]
+            }
+          },
+          'sass-loader'
+        ],
+      },
```
Comment (gabesullice) [inline webpack.config.js:57]: LGTM?? 🤷‍♂ 

😂

**Review by gabesullice — APPROVED** [review]
This is a great start! I'm really excited to see this fleshed out!

File: src/index.js (hunk 1 above)
Comment (zrpnr) [reply src/index.js:3]: Not directly- I was going to come back to that when/if we want to change its theme

File: webpack.config.js (hunk 2 above)
Comment (zrpnr) [reply webpack.config.js:57]: IKR :) this is a processing order- does sass, postcss and css, and finally splits it off into a build file.

**Pull request merged**